shaky/src/main/res/values/shaky_strings.xml

```diff
@@ -63,4 +63,11 @@
 
     <string name="shaky_setting">Settings</string>
     <string name="shaky_general_ok_string">OK</string>
+
+    <string name="shaky_select_subcategory">Select a subcategory</string>
+    <string-array name="shaky_bug_subcategories">
+        <item>Crash</item>
+        <item>Non-Fatal</item>
```

Comment:
Not sure if Non-Fatal is common enough to use it externally.
Isn't non-fatal just a regular bug?

Reply:
Changed to a more friendly string